### PR TITLE
Tnodb provision

### PR DESCRIPTION
**Pull request by zaibon**
+ update http provision source to match tnodb change

### CODE REVIEW

File: tools/tnodb_mock/provision.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		return
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
+	ch := make(chan []*provision.Reservation)
+
+	// start long polling
+	go func(ctx context.Context, ch chan<- []*provision.Reservation) {
+		for {
```
Comment (muhamadazmy) [inline tools/tnodb_mock/provision.go]: `defer close(ch)`

File: tools/tnodb_mock/provision.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		http.Error(w, fmt.Sprintf("node %s not found", nodeID), http.StatusNotModified)
+		return
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
+	ch := make(chan []*provision.Reservation)
+
+	// start long polling
+	go func(ctx context.Context, ch chan<- []*provision.Reservation) {
```
Comment (muhamadazmy) [inline tools/tnodb_mock/provision.go]: I don't get why this has to be in a go routine, it can be just a busy loop with intermittent sleeps

**Review by muhamadazmy — COMMENTED** [review]
I also don't get where the methods in tnodb/network.go are used?!

File: tools/tnodb_mock/provision.go (hunk 2 above)
Comment (zaibon) [reply tools/tnodb_mock/provision.go]: Indeed, each request is already executing in its own goroutine. thx

**Comment by zaibon** [comment]
> I also don't get where the methods in tnodb/network.go are used?!

They were all in the main.go before I just spitted the handlers into separate files

**Review by muhamadazmy — APPROVED** [review]

**Pull request merged**